test/instrumentation/modules/pg/knex.js

```diff
@@ -13,6 +13,11 @@ var agent = require('../../../..').start({
 var knexVersion = require('knex/package').version
 var semver = require('semver')
 
+if (semver.gte(knexVersion, '0.21.0') && semver.lt(process.version, '10.0.0')) {
+  // Skip these tests: knex@0.21.x dropped support for node v8.
+  process.exit(0)
+}
+
```

Comment:
I see a conditional that will exit with a success code of `0` if the knexVersion is greater than or equal to 0.21, and if the node version is less than 10. 👍

Reply:
If we switched (someday) from `tape` to `tap` we could use its more reliable TAP `skip()` handling to get actual TAP output that showed a message about skipping tests.  But obviously that's for another day.